README.md

```diff
@@ -145,6 +145,20 @@ cmake -DCMAKE_C_COMPILER=/usr/local/bin/gcc8 -DCMAKE_CXX_COMPILER=/usr/local/bin
 make -j$(sysctl -n hw.ncpu)
 ```
 </details>
+<details><summary>Haiku</summary>
+
+### Installing dependencies
+```
+pkgman install cmake devel:libsdl2 devel:libSDL2_mixer devel:libSDL2_ttf devel:libsodium

```

Comment:
A fresh nightly appears to want the names in lowercase
```suggestion
pkgman install cmake devel:libsdl2 devel:libsdl2_mixer devel:libsdl2_ttf devel:libsodium
```

Reply:
Of course it is. Whoops.

Reply:
Also, looks like 32 bit Haiku puts "_x86" after package names. Something to do with using different versions of gcc for BeOS compatibility...